### PR TITLE
PR for this week's homework

### PR DESCRIPTION
**Pull request by dgorden**
There was a bit of confusion with my previous PR. I created a PR from my repo to the main CC repo, which was not correct. Submitting this updated PR with the correct workflow.

### CODE REVIEW

File: 01_Core/week02/submissions/names.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+var names = "Bobby Craig, Dillon Gorden, Mark Kroh"
+
+function compare(nameOne,nameTwo) {
```
Comment (broffis) [inline 01_Core/week02/submissions/names.js:4]: I like the way you wrote this function. Good use of splitting the names into arrays and sorting based on that. Good use of the ES6 sort method as well!

**Review by broffis — APPROVED** [review]

File: 01_Core/week02/submissions/bullets.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Rainbow</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="navbar">
```
Comment (broffis) [inline 01_Core/week02/submissions/bullets.html]: There are tags for navigation in semantic html that would work better for this section

File: 01_Core/week02/submissions/bullets.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        <a href="index.html">Home</a>
+        <a href="page2.html">Form</a>   
+        <a href="bullets.html">Rainbow</a>
+        <a href="lastname.html">Sort By</a>
+        <div class="dropdown">
+            <a href="#hover">Hover</a>
```
Comment (broffis) [inline 01_Core/week02/submissions/bullets.html]: For drop downs, it's better to use `<li>` wrapped in an unordered list. You should combine that with the `<a>` tags you're already using

File: 01_Core/week02/submissions/bullets.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                <div class="content">
+                    <a href="">One</a>
+                    <a href="">Two</a>
+                    <a href="">Three</a>
+                </div>
+        </div>
+    </div>
+
+    <div id="sidebar">
+        <nav>
```
Comment (broffis) [inline 01_Core/week02/submissions/bullets.html:36]: Good use of the semantic `<nav>` element and an unordered list for navigation

File: 01_Core/week02/submissions/bullets.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Rainbow</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="navbar">
+        <a href="index.html">Home</a>
```
Comment (broffis) [inline 01_Core/week02/submissions/bullets.html]: Navigation lists should are lists and should be set up as such. Your sidebar makes use of this syntax

File: 01_Core/week02/submissions/index.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                <li><a href="https://highereducation.com">Higher Education</a></li>
+            </ul>
+        </nav>
+    </div>
+
+    <div class="centered-content">
+        <h1>Headline</h1>
+        <h2>Subhead 1</h2>
+        <h2>Subhead 2</h2>
+    <footer>
```
Comment (broffis) [inline 01_Core/week02/submissions/index.html]: Same thing here. Use the semantic `<nav>` element and an unordered list for navigation purposes

File: 01_Core/week02/submissions/lastname.html [hunk 7]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Last Name</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="navbar">
+        <a href="index.html">Home</a>
```
Comment (broffis) [inline 01_Core/week02/submissions/lastname.html]: Same as above, make sure to use the semantic nav elements and an unordered list for navigation

File: 01_Core/week02/submissions/lastname.html [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+        <a href="page2.html">Form</a>   
+        <a href="bullets.html">Rainbow</a>
+        <a href="lastname.html">Sort By</a>
+        <div class="dropdown">
```
Comment (broffis) [inline 01_Core/week02/submissions/lastname.html]: Dropdown lists should be unordered lists within a list element

File: 01_Core/week02/submissions/lastname.html [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+                <li><a href="https://redventures.com">Red Ventures</a></li>
+                <li><a href="https://bankrate.com">Bankrate</a></li>
+                <li><a href="https://highereducation.com">Higher Education</a></li>
+            </ul>
+        </nav>
+    </div>
+
+    <h1>Please enter your name:</h1>
+    <form>
+        <label for="subject" >Name</label>
```
Comment (broffis) [inline 01_Core/week02/submissions/lastname.html]: The for="" should relate back to the id (`#`) that the label is for

File: 01_Core/week02/submissions/page2.html [hunk 10]
```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+        <meta charset="UTF-8">
+        <title>Form</title>
+        <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+    <div class="navbar">
```
Comment (broffis) [inline 01_Core/week02/submissions/page2.html]: Same as above, make sure to use the semantic nav elements and an unordered list for navigation

File: 01_Core/week02/submissions/page2.html [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+        <a href="index.html">Home</a>
+        <a href="page2.html">Form</a>   
+        <a href="bullets.html">Rainbow</a>
+        <a href="lastname.html">Sort By</a>
+        <div class="dropdown">
```
Comment (broffis) [inline 01_Core/week02/submissions/page2.html]: Same as above, make sure to use lists for drop downs and all other navigation

File: 01_Core/week02/submissions/page2.html [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+                    <option value="waterPolo">Water Polo</option>
+                </select>
+            </div>
+        
+            <button>Save</button>
+        </form>
+    </div>
+
+    <footer>
+        <div class="sidenav">
```
Comment (broffis) [inline 01_Core/week02/submissions/page2.html:79]: Make sure to convert this to a list element as well

File: 01_Core/week02/submissions/bullets.html [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+            <div class="sidenav">
+                <a href="https://www.facebook.com">Facebook</a>
+                <a href="https://www.twitter.com">Twitter</a>
+                <a href="https://www.instagram.com">Instagram</a>
+            </div>
+            <span>&copy; Copyright 2019 </span>  
+        </footer> 
+    </div> 
+</body>
+</html>
```
Comment (broffis) [inline 01_Core/week02/submissions/bullets.html]: Something our linters will call out, but I never really understood is ending a file with a new line. It's something you should always do. If you want to read why, https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: 01_Core/week02/submissions/index.html [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+            <a href="https://www.facebook.com">Facebook</a>
+            <a href="https://www.twitter.com">Twitter</a>
+            <a href="https://www.instagram.com">Instagram</a>
+        </div>
+            &copy; Copyright 2019   
+    </footer> 
+    </div>
+ 
+</body>
+</html>
```
Comment (broffis) [inline 01_Core/week02/submissions/index.html]: Same as above. Files should always end with an empty line

File: 01_Core/week02/submissions/lastname.html [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+            <a href="https://www.instagram.com">Instagram</a>
+        </div>
+            <span>&copy; Copyright 2019 </span>  
+    </footer> 
+    
+
+
+<script src="names.js"></script> 
+</body>
+</html>
```
Comment (broffis) [inline 01_Core/week02/submissions/lastname.html]: Files should always end with an empty line

File: 01_Core/week02/submissions/page2.html [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+            <a href="https://www.facebook.com">Facebook</a>
+            <a href="https://www.twitter.com">Twitter</a>
+            <a href="https://www.instagram.com">Instagram</a>
+        </div>
+            &copy; Copyright 2019   
+    </footer>
+
+</body>
+</html>
```
Comment (broffis) [inline 01_Core/week02/submissions/page2.html]: Make sure you end every file with an empty line

File: 01_Core/week02/submissions/styles.css [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+    min-width: 200px;
+}
+
+.content a {
+    display: block;
+}
+
+.dropdown:hover .content {
+    display: block;
+}
```
Comment (broffis) [inline 01_Core/week02/submissions/styles.css:54]: Make sure you're ending each file with an empty line

File: 01_Core/week02/submissions/styles.css [hunk 18]
```diff
@@ -0,0 +1,49 @@
+h1 {
```
Comment (broffis) [inline 01_Core/week02/submissions/styles.css:1]: Make sure to include a CSS reset file as well. These help normalize styles between different web browsers
https://meyerweb.com/eric/tools/css/reset/

File: 01_Core/week02/submissions/page2.html [hunk 19]
```diff
@@ -0,0 +1,78 @@
...
+                <li><a href="https://redventures.com">Red Ventures</a></li>
+                <li><a href="https://bankrate.com">Bankrate</a></li>
+                <li><a href="https://highereducation.com">Higher Education</a></li>
+            </ul>
+        </nav>
+    </div>
+
+    <div class="form">
+        <h3>Please complete the following:</h3>
+        <form method="GET">
```
Comment (broffis) [inline 01_Core/week02/submissions/page2.html]: GET should be lowercase in method calls

File: 01_Core/week02/submissions/bullets.html (hunk 2 above)
Comment (dgorden) [reply 01_Core/week02/submissions/bullets.html]: tags have been added

File: 01_Core/week02/submissions/bullets.html (hunk 5 above)
Comment (dgorden) [reply 01_Core/week02/submissions/bullets.html]: set up nav lists

File: 01_Core/week02/submissions/bullets.html (hunk 3 above)
Comment (dgorden) [reply 01_Core/week02/submissions/bullets.html]: wrapped <a> with <li>

File: 01_Core/week02/submissions/bullets.html (hunk 13 above)
Comment (dgorden) [reply 01_Core/week02/submissions/bullets.html]: added empty line to each file

File: 01_Core/week02/submissions/bullets.html [hunk 20]
```diff
@@ -0,0 +1,68 @@
...
+                <a href="https://www.facebook.com">Facebook</a>
+                <a href="https://www.twitter.com">Twitter</a>
+                <a href="https://www.instagram.com">Instagram</a>
+            </div>
+            <span>&copy; Copyright 2019 </span>
+        </footer>
+    </div>
+</body>
+
+</html>
```
Comment (broffis) [inline 01_Core/week02/submissions/bullets.html]: Missing an empty line at the end of the file

File: 01_Core/week02/submissions/index.html [hunk 21]
```diff
@@ -0,0 +1,61 @@
...
+                <a href="https://www.twitter.com">Twitter</a>
+                <a href="https://www.instagram.com">Instagram</a>
+            </div>
+            &copy; Copyright 2019
+        </footer>
+    </div>
+
+</body>
+
+</html>
```
Comment (broffis) [inline 01_Core/week02/submissions/index.html]: Missing an empty line at the end of the file

File: 01_Core/week02/submissions/lastname.html [hunk 22]
```diff
@@ -0,0 +1,72 @@
...
+        </div>
+        <span>&copy; Copyright 2019 </span>
+    </footer>
+
+
+
+    <script src="names.js"></script>
+</body>
+
+</html>
```
Comment (broffis) [inline 01_Core/week02/submissions/lastname.html]: Missing an empty line at the end of the file

File: 01_Core/week02/submissions/normalize.css [hunk 23]
```diff
@@ -0,0 +1,349 @@
...
+    display: none;
+  }
+  
+  /**
+   * Add the correct display in IE 10.
+   */
+  
+  [hidden] {
+    display: none;
+  }
```
Comment (broffis) [inline 01_Core/week02/submissions/normalize.css:349]: Missing an empty line at the end of the file

File: 01_Core/week02/submissions/page2.html [hunk 24]
```diff
@@ -0,0 +1,89 @@
...
+            <a href="https://www.facebook.com">Facebook</a>
+            <a href="https://www.twitter.com">Twitter</a>
+            <a href="https://www.instagram.com">Instagram</a>
+        </div>
+        &copy; Copyright 2019
+    </footer>
+
+</body>
+
+</html>
```
Comment (broffis) [inline 01_Core/week02/submissions/page2.html]: Missing an empty line at the end of the file

File: 01_Core/week02/submissions/styles.css [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
+    min-width: 200px;
+}
+
+.content li {
+    display: block;
+}
+
+.dropdown:hover .content {
+    display: block;
+}
```
Comment (broffis) [inline 01_Core/week02/submissions/styles.css:54]: Missing an empty line at the end of the file

**Comment by broffis** [comment]
You probably don't need both a normalize.css and a reset.css as they generally do the same thing

**Review by broffis — APPROVED** [review]

**Pull request merged**